tests/queries/0_stateless/02294_decimal_second_errors.sql

```diff
@@ -0,0 +1,11 @@
+SELECT 1 SETTINGS max_execution_time=NaN; -- { serverError 72 }
+SELECT 1 SETTINGS max_execution_time=Infinity; -- { serverError 72 };
+SELECT 1 SETTINGS max_execution_time=-Infinity; -- { serverError 72 };
+
+-- Ok values
+SELECT 1 SETTINGS max_execution_time=-0.5;
+SELECT 1 SETTINGS max_execution_time=0.5;
```

Comment:
`DB::Exception: Timeout exceeded: elapsed 0.505827683 seconds, maximum: 0.5.` :( 
https://s3.amazonaws.com/clickhouse-test-reports/39507/fe1e326701b0566881dc7d51793125a911faa07e/stateless_tests__tsan__[2_5].html

Reply:
I'll raise it. There are other test already covering 0.something so we don't need to keep a subsecond value here